apps/dot-voting/package.json

```diff
@@ -13,9 +13,7 @@
     "frontend": "npm run sync-assets && parcel app/index.html --port 4444",
     "ganache-cli:test": "sh ../../node_modules/@aragon/test-helpers/ganache-cli.sh",
     "lint": "solium --dir ./contracts",
-    "postcoverage": " sed -i 's+./IForwarder.sol+@aragon/os/contracts/common/IForwarder.sol+g' ../shared/test-helpers/contracts/common/ADynamicForwarder.sol",
     "precommit": "lint-staged",
-    "precoverage": "sed -i 's+@aragon/os/contracts/common/IForwarder.sol+./IForwarder.sol+g' ../shared/test-helpers/contracts/common/ADynamicForwarder.sol",
```

Comment:
It doesn't make sense to remove this since addresses a bug in the coverage tool. We should upgrade our coverage tool before removing this since it doesn't make sense to create our own duplicated import when it already exists in a dependency we're using

Reply:
If you take the time to test the code you will notice I fixed the coverage bug so there is no need to use these hooks anymore.

I commented about it in the commit message:
`- Removing pre and post coverage hooks, unnecessary for validation after
inlining the interface`